### ISSUE TITLE
"Cannot find module" in VSCode

### ISSUE THREAD

**Issue by finnsnape** [janosh/svicons#9]
Getting `"Cannot find module '@svicons/ionicons-solid/trophy.svelte' or its corresponding type declarations.ts(2307)"` on:
```html
<script lang="ts">
  import Trophy from '@svicons/ionicons-solid/trophy.svelte';
</script>
```
In VSCode despite the icon actually working when doing `<Trophy/>`. I installed it with `npm i -D @svicons/ionicons-solid`.

Not sure if this is something you can fix, but perhaps it's an issue with the way you're exporting the icons in the js/ts files.

**Comment by janosh** [comment]
Thanks for the report. I noticed that too. I think it started a week or 2 ago. Looks like Svelte now expects `foo.d.ts` type declaration files to be named `foo.svelte.d.ts`. Patch release incoming.

**Comment by janosh** [comment]
Fixed in [v0.1.9](https://www.npmjs.com/package/@svicons/ionicons-solid/v/0.1.9).

**Comment by finnsnape** [comment]
> I missed something. Fixed in [v0.1.11](https://www.npmjs.com/package/@svicons/ionicons-solid/v/0.1.11).

Works great after updating, thanks
